src/no/runsafe/UserControl/database/PlayerKickLog.java

```diff
@@ -22,22 +25,39 @@ public ISchemaUpdate getSchemaUpdateQueries()
 		update.addQueries(
 			"CREATE TABLE player_kick_log (" +
 				"`name` varchar(255) NOT NULL," +
-				"`timestamp` datetime NULL," +
+				"`timestamp` datetime NOT NULL," +

```

Comment:
that does make sense, because a null value in a unique constraint violates the relational model. so the question becomes; how come this was there..?
The only sensible explanations I can come up with on the fly are
 A) MySQL let you violate this in an earlier version
 B) the schema was manually created on our server
 C) the CREATE TABLE query has been changed since the initial build